test-at/src/test/scala/com/stratio/sparkta/testa/ISocketOParquetRawAT.scala

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (C) 2015 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.testa
+
+import scala.reflect.io.File
+
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.SQLContext
+import org.json4s._
+import org.json4s.jackson.JsonMethods._
+
+import com.stratio.sparkta.driver.dto.AggregationPoliciesDto
+import com.stratio.sparkta.driver.factory.SparkContextFactory
+import com.stratio.sparkta.sdk.JsoneyStringSerializer
+
+class ISocketOParquetRawAT extends SparktaATSuite {
+
+  val PolicyEndSleep = 30000
+  val NumExecutors = 4
+  val Policy = getClass.getClassLoader.getResource("policies/ISocket-OParquetRaw.json")
+  val PathToPolicy = Policy.getPath
+  val PathToCsv = getClass.getClassLoader.getResource("fixtures/ISocket-OParquetRaw.csv").getPath
+
+  implicit val formats = DefaultFormats + new JsoneyStringSerializer()
+  val parquetPath = parse(Policy.openStream()).extract[AggregationPoliciesDto].rawDataParquetPath
+
+  before {
+    zookeeperStart
+    socketStart
+  }
+
+  after {
+    serverSocket.close()
+    zkTestServer.stop()
+    File(parquetPath).deleteRecursively
+  }
+
+  val Total = 16
+
+  "Sparkta" should {
+
+    "start in 3 seconds on 9090" in {
+      startSparkta
```

Comment:
Pls update the test name and description, I think that we're doing much more than start SpaRkTA in 3 seconds.